brickstrap.sh

```diff
@@ -461,14 +517,18 @@ function brp_configure_packages () {
         fi
     done
 
-    # run dpkg `--configure -a` twice because of errors during the first run
+    brp_fixup_etc_shells
+
     info "configuring packages..."
-    br_chroot_bind /usr/bin/dpkg --configure -a || \
-    br_chroot_bind /usr/bin/dpkg --configure -a || true
+    br_chroot_bind /usr/bin/dpkg --configure -a
+
+    export PATH="$BRP_OLD_PATH"
 
     # remove our temporary awk symlink as it is no longer required.
-    info "Removing awk temporary symlink"
-    br_chroot rm -f /usr/local/bin/awk
+    if [ -x "/$(br_chroot_brp_dir)/bin/awk" ]; then
```

Comment:
I wanted it out of `/usr/local/bin` because `/usr` *should* not be messed with if at all possible. And this is quite possible.

That is precisely the reason we have the `/brickstrap`: for our temporary modifications to the rootfs and to hide those modifications from other code/tars/imaging where possible.

So I disagree: much better to locally redefine PATH for a short duration than to root around `/usr/local` which is for the configuration author to deploy their own apps to if these aren't package properly yet. (Following the somewhat dated principle of: /usr -> package manager; /usr/local -> sysadmin.)